rupture-models/20090715-dusky-sound/Beavan_EtAl2010/CP1/3124785xCP1.csv

```diff
@@ -0,0 +1,426 @@
+SEGMENT,LAT,LON,DEPTH,SLIP,STRIKE,DIP,RAKE,WIDTH,LENGTH
```

Comment:
Question: shall we keep the same header content and fiels in all the models ?
as in #734 `SEGMENT,lat,lon,NORTHING,EASTING,DEPTH,SLIP,STRIKE,DIP,RAKE,WIDTH,LENGTH,RUPTURE TIME`
and leave  e.g RUPTURE TIME  empty as in`,,`


Although  ther are maybe too many slight differences between the slip models

Reply:
Yes, I agree that we have the same header fields in all the .csv files. I will do that.

Reply:
We now have same header fields in the csv files.